subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/downstream/test/TestDownstreamPlugin.java

```diff
@@ -22,12 +27,30 @@
 
     public void apply(final Project project) {
         IncubatingWarning.warn("downstream-testing plugin");
-        TaskMaker.task(project, TEST_DOWNSTREAM_TASK, TestDownstreamTask.class, new Action<TestDownstreamTask>() {
+
+        project.getPlugins().apply(UploadGistsPlugin.class);
```

Comment:
Can you update the javadoc of this class? We're changing the behavior and applying another plugin.